main.go

```diff
@@ -777,6 +786,44 @@ func getFreeLocalAddr() (string, error) {
 	return fmt.Sprintf("127.0.0.1:%d", l.Addr().(*net.TCPAddr).Port), nil
 }
 
+// createOcmNamespace create open-cluster-management-policies namespace
+// on the managedClusters and hubCluster for standalone policies
+func createOcmNamespace(mgrCtx context.Context, managedClient *rest.Config,
```

Comment:
Could you please use the addon controller instead like we do for the cluster namespace on the managed cluster?
https://github.com/open-cluster-management-io/governance-policy-addon-controller/blob/main/pkg/addon/policyframework/manifests/managedclusterchart/templates/namespace.yaml